src/InteractsWithMedia.php

```diff
@@ -289,24 +289,32 @@ public function getMediaModel(): string
         return config('media-library.media_model');
     }
 
+    private function getMediaItem(string $collectionName, $filters, string $firstOrLast)
```

Comment:
Instead of a string, introduce a new enum called `CollectionPosition`, with `first` and `last` cases.